lab-matt/route/model-route.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+
+const Model = require('../model/model');
+const response = require('../lib/response');
+const Storage = require('../lib/storage');
+const newStorage = new Storage(`${__dirname}/../data`);
+
+
+module.exports = function(router){
+  router.post('/api/model/create', function(req, res){
```

Comment:
This API should only have one endpoint, '/api/model', here you have a particular endpoint for each route, when really you should have one endpoint for all three.  The server will recognize the different request methods and act accordingly.